testsuite/fix-versions.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-oldversion=0.2
-newversion=0.3
+oldversion=0.4
+newversion=0.1
```

Comment:
Why going to version `0.1` instead of `0.5`?

Reply:
To avoid confusing potential crate contributors I when to `tmp-0.1` instead of `devel-0.x` for the migrated community index. In any case once the PRs are to be merged I planned to use the proper `devel-0.x` one. I forgot to communicate this.

Reply:
ok :+1: